protocol-units/settlement/mcr/client/src/eth/mod.rs

```diff
@@ -1,2 +1,3 @@
 pub mod mcr;
 // pub mod staking;
+pub mod utils;
```

Comment:
This is dead code and does not refer to anything yet.

Reply:
yes remove it.